### PR TITLE
Alert feature

### PR DESCRIPTION
**Pull request by tusharkandpal**
- added alert component
- styling card.css
- some fixes

Recording 2022-02-02 at 21 24 59

### CODE REVIEW

**Comment by krishnakant01** [comment]
The code looks clean! You are good to go.
Deploy it on netlify/ vercel, it will be easier for pr reviews.

**Review by rushikesh3299 — COMMENTED** [review]
code has been written very clean. Everything looks fine to me. You can merge this.

File: components/alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    --danger-alert-light: #F8D7DA;
+    --warning-alert-light: #FFF3CD;
+    --info-alert-light: #CFF4FC;
+    --primary-alert-dark: #2864be;
+    --secondary-alert-dark: #808080;
+    --success-alert-dark: #17925a;
+    --danger-alert-dark: #ca2b39;
+    --warning-alert-dark: #b89628;
+    --info-alert-dark: #2d96ad;
+}
```
Comment (JantuDeb) [inline components/alert/alert.css:22]: Put all the variables in one CSS file, it should be your main CSS file for CL

**Review by Raghav888 — APPROVED** [review]
Seems perfect

**Review by JantuDeb — CHANGES_REQUESTED** [review]
suggested some changes.

File: components/alert/alert.css (hunk 1 above)
Comment (tusharkandpal) [reply components/alert/alert.css:22]: i'm going to separate variables based on component so this is the approach i prefer @JantuDeb

Comment (krishnakant01) [reply components/alert/alert.css:22]: I think using all variables in 1 file will reduce redundancies but this works too

**Comment by krishnakant01** [comment]
UI is really good! just make your sidebar sticky or fixed and you're good to go!

**Review by krishnakant01 — APPROVED** [review]

**Comment by tusharkandpal** [comment]
> UI is really good! just make your sidebar sticky or fixed and you're good to go!

ya that will be fixed automatically once i merge all the changes

**Pull request merged**